### ISSUE TITLE
Make new search bar more accessible

### ISSUE THREAD

**Issue by RandellDawson** [freeCodeCamp/freeCodeCamp#36693]
If I type in something and results appear, it would be nice to be able to use the up and down arrow keys to navigate through the results and then press Enter to make a selection.

**Comment by Zannisdf** [comment]
Can I take a look at this one?

**Comment by RandellDawson** [comment]
I believe @scissorsneedfoodtoo may be already working on a fix for this?

**Comment by scissorsneedfoodtoo** [comment]
No, I haven't started on this yet. Please go ahead, @Zannisdf.

**Comment by scissorsneedfoodtoo** [comment]
Another necessary fix:
* Users should be able to type into the search bar and press enter to search for a term.

Nice to haves:
* Gmail-like hotkey support--Pressing `/` will focus on the search bar, and `j` and `k` keys can act as up or down arrow keys.
